### ISSUE TITLE
Unordered list candidate voorbereidingsfase toegankelijkheid

### ISSUE THREAD

**Issue by RenateRoke** [nl-design-system/candidate#948]
## TLDR

De accessibility voorbereidingsfase van Unordered list moet gedaan worden om die te kunnen maken. De stappen zijn beschreven in het Candidate stappenplan en het Unordered list document in Slack.

## Actiepunten

- [ ] Analyse van bestaande community componenten
- [ ] Beschrijven van de gewenste semantiek in verschillende varianten
- [ ] Samenstellen van de succescriteria die relevant zijn voor de component
- [ ] Succescriteria onderverdelen in component en contextuele succescriteria
- [ ] Acceptatiecriteria opstellen
- [ ] Testcases uitwerken
- [ ] Unittest voorstel doen

## Acceptatiecriteria

- [ ] De voorbereidingsfase van Unordered list is afgerond
